tests/Functional/Fixtures/web/symfony.php

```diff
@@ -21,4 +21,4 @@
 $request = Request::createFromGlobals();
 $response = $kernel->handle($request);
 $response->send();
-$kernel->terminate($request, $response);
+//$kernel->terminate($request, $response);
```

Comment:
it makes the tests green, but only because the response is sent before hhvm crashes... maybe we do not need to bother for FOSHttpCache, if we do not rely on any kernel terminate events.